packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -223,10 +235,20 @@ export const CMReleasesContainer = () => {
    * - Content types without draft and publish cannot add entries to release
    * TODO v5: All contentTypes will have draft and publish enabled
    */
-  if (isCreatingEntry || !contentType.options?.draftAndPublish) {
+  if (isCreatingEntry || !contentType?.options?.draftAndPublish) {
     return null;
   }
 
+  const toggleAddActionToReleaseModal = () => setShowModal((prev) => !prev);
+
+  const getReleaseColorVariant = (actionType: 'publish' | 'unpublish') => {
```

Comment:
maybe you could do the string concatenation in this function? With a new param like "shade"